packages/core/package-lock.json

```diff
@@ -1,555 +0,0 @@
-{
```

Comment:
@tatomyr 
This package-lock looks obsolete in the mono-repo setup, but please tell me if there is some idea behind it.

Reply:
You can try removing it and see what happens.